save-agent/build.gradle.kts

```diff
@@ -30,13 +30,39 @@ kotlin {
         }
     }
 
+    // generate kotlin file with project version
+    val generateVersionFileTaskProvider = tasks.register("generateVersionFile") {
```

Comment:
do already we have this logic in save-cloud somewhere? may be we can create some method, which will expect few arguments like `internal const val AGENT_VERSION = "$version"` etc

Reply:
or we have something similar in save-cli?

Reply:
> do already we have this logic in save-cloud somewhere? may be we can create some method, which will expect few arguments like internal const val AGENT_VERSION = "$version" etc

Yes, we have similar logic with frontend

Reply:
moved to a common place